.devcontainer/devcontainer.json

```diff
@@ -13,6 +13,7 @@
         "eamodio.gitlens",
         "vadimcn.vscode-lldb",
         "rust-lang.rust-analyzer",
+        "ms-azuretools.vscode-docker",
```

Comment:
added this because I always seem to be editing the docker file in the container and vscode keeps asking if I want to install it.